projects/gallery/src/lib/components/gallery/gallery.component.ts

```diff
@@ -86,6 +87,12 @@ export class GalleryComponent {
     );
   }
 
+  @HostBinding('class.rtl')
+  get _isRtl() {
+    return(this.isRtl);
```

Comment:
Can you remove the parentheses? I know it works, it's just a bit strange syntax.

Reply:
In my IDE I receive a ts error if I try to remove the parentheses

Reply:
I think you might have some linter setting there which makes it look more like c++ or smth. Both syntaxes are valid JS.